Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -612,6 +612,8 @@
       - Truncheon
       - VoiceTrigger
       - WeaponAdvancedLaser
+      - WeaponDisablerPractice
```

Comment:
Why is WeaponDisablerPractice duplicated here? Its already under static recipes and there is no mention of this on the PR

Reply:
This duplication of WeaponDisablerPractice is still here. Is this intentional?